test/passing/error4.ml.opts

```diff
@@ -1 +1 @@
---no-comment-check
\ No newline at end of file
+--no-comment-check=true
```

Comment:
Is it possible to not have to change this? Being required to specify true or false is an uncommon interface.